### DISCUSSION TITLE
stoicism/amor_fati/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#9] [General]
# stoicism/amor_fati/

https://shanenull.com/stoicism/amor_fati/

**Comment by shane0** [comment]
# enjoy life despite conditions

- sid smith step 1 and 2
- let go of delusions
- accept the unpredictable
- build something better than this

https://youtube.com/clip/Ugkxbxup6ZKGhctmWSSGQ-9KswUdHzMbVMsE?si=_gmkoAqje3KvtiqI

**Comment by shane0** [comment]
# natural catastrophes

- mt baker is dangerous https://youtu.be/eNTBML_AKLc

**Reply by shane0** [reply]
https://www.seattletimes.com/seattle-news/washington-volcanoes-remain-among-the-most-dangerous-in-the-country-new-report-says/#:~:text=USGS's%20National%20Volcanic%20Threat%20Assessment,volcanoes%20are%20scored%20and%20ranked.

**Reply by shane0** [reply]
| Volcano | VAI Level (Historical) | Notable Features |
|---|---|---|
| Mount St. Helens | 5 (1980 eruption) | Known for its catastrophic eruption |
| Mount Rainier | 3-4 | The most prominent peak in the Cascades |
| Mount Baker | 3 | Located in the northern Cascades |
| Mount Adams | 2-3 | Relatively quiet history, but still a threat |
| Glacier Peak | 4 | Remote location, but capable of large eruptions |
